docs/serving/autoscaling/scale-bounds.md

```diff
@@ -217,4 +215,58 @@ delay period.
         autoscaler:
           scale-down-delay: "15m"
     ```
+## Stable window
+
+The stable window defines the sliding time window over which metrics are averaged to provide the input for scaling decisions when the autoscaler is not in [Panic mode](kpa-specific.md).
+
+* **Global key:** `stable-window`
+* **Per-revision annotation key:** `autoscaling.knative.dev/window`
+* **Possible values:** Duration, `6s` <= value <= `1h`
+* **Default:** `60s`
+
+!!! note
+    During scale down, the last Replica will only be removed after there has not been any traffic to the Revision for the entire duration of the stable window.
+
+**Example:**
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            autoscaling.knative.dev/window: "40s"
```

Comment:
I think the annotation is on the Service Resource not on the pod spec right?

Reply:
@csantanapr do you have a suggested change that should be made here? I'd like to get this PR merged.

Reply:
>I think the annotation is on the Service Resource not on the pod spec right?

I double-check, I don't know what I was thinking when I wrote the comment

The annotation is in the correct space under `spec.metadata.annotations` 
Ignore my comment, the way you have it is correct.